### DISCUSSION TITLE
Episode 140: 🇨🇦 CppNorth Live! 🇨🇦 Victor Cuira, Andreas Weis & More!

### DISCUSSION THREAD

**Discussion by codereport** [codereport/adsp2#31] [Episode]
Episode Link: https://adspthepodcast.com/2023/07/28/Episode-140.html

Feel free to write comments or ask questions 🙂

**Comment by vmiheer** [comment]
This should come with huge "SPOILER ALERT" warning for poor folks like us who didn't see all the talks at CppNorth and now can't wait to see them on YouTube. ;).
